src/ui/ControlTable/ControlTable.js

```diff
@@ -13,20 +13,23 @@ const ControlTable = (props) => {
     if (Array.isArray(props.data) && props.data.length > 0) {
       const newColumns = generateColumns(props.data, props.totalColCriteria, props.totalRowCriteria);
       const newRows = generateRows(props.data, props.totalColCriteria, props.totalRowCriteria);
-      console.log(newRows);
       setColumns(newColumns);
       setRows(newRows);
     }
+    else {
+      setColumns([]);
+      setRows([]);
+    }
```

Comment:
Why? doesn't generateColumns() and generateRows() not return [] when there is no data?